mne/io/egi/egimff.py

```diff
@@ -907,6 +909,142 @@ def _read_evoked_mff(fname, condition, channel_naming='E%d', verbose=None):
                        nave=nave, verbose=verbose)
 
 
+@fill_doc
+def export_evokeds_to_mff(fname, evoked, device, history):
```

Comment:
this function should be private.

export_evokeds will be AFAIK the public entry point to the API.